crates/core/src/syn/parser/stmt/define.rs

```diff
@@ -1444,6 +1446,27 @@ impl Parser<'_> {
 		})
 	}
 
+	pub(crate) async fn parse_defaults_config(
+		&mut self,
+		stk: &mut Stk,
+	) -> ParseResult<DefaultsConfig> {
+		let mut config = DefaultsConfig::default();
+		loop {
+			match self.peek_kind() {
+				t!("NAMESPACE") => {
+					self.pop_peek();
+					config.namespace = Some(stk.run(|stk| self.parse_expr_field(stk)).await?);
+				}
+				t!("DATABASE") => {
+					self.pop_peek();
+					config.database = Some(stk.run(|stk| self.parse_expr_field(stk)).await?);
+				}
+				_ => break,
+			}
+		}
+		Ok(config)
```

Comment:
This is able to parse nothing: `DEFINE CONFIG DEFAULT;` We should probably force a define statement to at least do something.